### ISSUE TITLE
Work requirements due date

### ISSUE THREAD

**Issue by MichalisKas** [FreedomCoop/valuenetwork#414]
If you use a recipe to create a process, although the due date of the output is set right, the due dates of the work requirements are set to the starting one rather than the last (as they should).
![screenshot-2017-12-19 open collaborative platform process ce area facilitation to january 2017 michalis starting 2018-01-](https://user-images.githubusercontent.com/20679274/34177401-21e239f6-e50c-11e7-8f3e-611579b1ff04.png)

**Comment by bhaugen** [comment]
@MichalisKas good catch. I'll get this fixed, but we got a lot of family activity over the holidays, so I don't know when. But I'll ping you.

This is not an excuse, but when I coded that a few years ago, I was following MRP logic, which focuses on material requirements, which usually need to be available at the beginning of a process. Work requirements, as you wrote, are different, and I did not differentiate.
